packages/dscc-scripts/test/viz/util.spec.ts

```diff
@@ -101,3 +101,31 @@ test('invalid config', () => {
     expect(() => sut.validateConfig(configContents)).toThrow('Invalid config');
   });
 });
+
+describe('manifest validation', () => {
+  test('devMode is a boolean', () => {
+    const manifest = JSON.stringify({
```

Comment:
Where is devMode in this sample manifest?

Reply:
Oh, looks like I undo-ed it. Will add back in.